Source/Parser.swift

```diff
@@ -281,7 +191,7 @@ public class Parser {
         return parsed
     }
 
-    private class func parsePropertiesForJSONArray(_ array: [Any], closure: (ParserPropertyMaker) -> Void) throws -> [String: Any] {
+    private class func parseEntityForJSONArray(_ array: [Any], closure: (ParserPropertyMaker) -> Void) throws -> [String: Any] {
```

Comment:
`parseEntity(forJSONArray:)`? I don't like the `array` being in the signature, but I think it's an ambiguity problem if you don't have it.